src/storage.ts

```diff
@@ -15,9 +15,14 @@
  */
 
 import { openDB, DBSchema, IDBPDatabase } from 'idb';
+import { v4 as uuidv4 } from 'uuid';
```

Comment:
What is the bundle size increase we're seeing by including this dependency?

Reply:
`llaminator.<hash>.js`: 81,100 B -> 82,589 B (~1.45 KiB, ~1.84% increase)
`sw.js`: 20,761 B -> 22,099 B (~1.31 KiB, ~6.44% increase)